minecraft_servers/crystopia/manifest.json

```diff
@@ -0,0 +1,28 @@
+{
+  "server_name": "crystopia",
+  "nice_name": "Crystopia",
+  "direct_ip": "play.crystopia.net",
+  "server_wildcards": [
+    "mc.crystopia.net"
+  ],
+  "social": {
+    "web": "https://crystopia.net",
+    "web_shop": "none",
+    "web_support": "https://crystopia.net/support",
+    "twitter": "CrystopiaNet",
+    "youtube": "https://www.youtube.com/@",
+    "discord": "https://dc.crystopia.net",
```

Comment:
Custom discord invite links are only supported for LabyMod partners at the moment.
Please use a discord invite link.

Documentation reference: https://github.com/LabyMod/server-media/blob/master/docs/Manifest.md#discord-url